my_list.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'Enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*items)
+    @list = items
+  end
+
+  def each
+    @list.each { |ele| yield ele }
+  end
```

Comment:
- [ ] Please try to avoid using the existing each method but instead loop over it yourself.